### ISSUE TITLE
Tree element actions not readable by screen reader (open editors, git, search)

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#8658]
VSCode Version: insiders 7/1
OS Version: Windows

Screenreader can not read Close Editor button in front of an editor in the explorer's Open Editor section.

**Comment by isidorn** [comment]
@bpasero looking at the split view header actionbar and the actionbar in the tree - they look exactly the same, they have the same role and some title. The split view header gets nicely read out, however the tree element action bar is left silently. Any hints?
Not sure if us modifying the visibilty screws up the screen reader (we only show the action on hover and focus)

**Comment by bpasero** [comment]
i have a bug for this, I think no action inside the tree gets read anywhere

**Comment by isidorn** [comment]
Closing as dup then
